code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -102,18 +103,26 @@
 			to_chat(src, "<span class='userdanger'>The heat from [I] cauterizes your bleeding!</span>")
 			playsound(src, 'sound/surgery/cautery2.ogg', 70)
 
-		//dismemberment
-		var/dismemberthreshold = (((affecting.max_damage * 2) / max(I.is_sharp(), 0.5)) - (affecting.get_damage() + ((I.w_class - 3) * 10) + ((I.attack_weight - 1) * 15)))
-		if(HAS_TRAIT(src, TRAIT_EASYDISMEMBER))
-			dismemberthreshold -= 50
-		if(I.is_sharp())
-			dismemberthreshold = min(((affecting.max_damage * 2) - affecting.get_damage()), dismemberthreshold) //makes it so limbs wont become immune to being dismembered if the item is sharp
-			if(stat == DEAD)
-				dismemberthreshold = dismemberthreshold / 3
-		if(I.force >= dismemberthreshold && I.force >= 10)
-			if(affecting.dismember(I.damtype))
-				I.add_mob_blood(src)
-				playsound(get_turf(src), I.get_dismember_sound(), 80, 1)
+		var/dismember_limb = FALSE
+		var/weapon_sharpness = I.is_sharp()
+
+		if((HAS_TRAIT(src, TRAIT_EASYDISMEMBER) && limb_damage) || weapon_sharpness == SHARP_DISMEMBER_EASY && prob(I.force))
```

Comment:
What is the intended order of operations here? If you have TRAIT_EASYDISMEMBER then you will have a 100% chance to lose limbs regardless of the force due to precedence.

With proper parenthesis, this expression looks like this:
`(HAS_TRAIT(src, TRAIT_EASYDISMEMBER) && limb_damage) || (weapon_sharpness == SHARP_DISMEMBER_EASY && prob(I.force))`

Reply:
Thank you for catching that, I overlooked it when I got tired of arguing about my older logic (before the PR was posted) being fine with dragon and just copied his to appease him because I didn't want it to block the PR when it was posted. 

I'll fix it up now - the correct order of operations is:
* If the mob has easy dismember and a damaged limb -> direct % chance equal to force to dismember.
* If the weapon has easy dismember -> direct % chance equal to force to dismember